src/NinjaTools.FluentMockServer.API/Proxy/Visitors/RequestMatcherEvaluationVisitor.cs

```diff
@@ -11,85 +12,80 @@
 namespace NinjaTools.FluentMockServer.API.Proxy.Visitors
 {
     /// <inheritdoc cref="IVisitor" />
-    [SuppressMessage("ReSharper", "ConvertIfStatementToReturnStatement")]
-    public sealed class ComparasionVisitor : IVisitor,
-        IVisitor<RequestMatcher>,
+    [DebuggerDisplay("{" + nameof(DebuggerDisplay) + "}")]
+    public sealed class RequestMatcherEvaluationVisitor :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on '_cts'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4928294)